References: https://github.com/google/highway/issues/2671
References: https://bugzilla.opensuse.org/show_bug.cgi?id=1248740

Deactivate AVX10.2. gcc 15.2 emits opcodes that are only understood
by binutils >=2.44, but Tumbleweed still has 2.43.

/tmp/ccVeFR1S.s: Assembler messages:
/tmp/ccVeFR1S.s:13780: Error: no such instruction: `vucomxss .LC9(%rip),%xmm1'

Furthermore, deactivate AVX3 (for Leap 15.7) because the compiler is
"too old" and does not have an instrinsic.

hwy/ops/x86_512-inl.h:1310:49: error: '_mm512_gf2p8affine_epi64_epi8'
was not declared in this scope
---
 hwy/detect_targets.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: highway-1.3.0/hwy/detect_targets.h
===================================================================
--- highway-1.3.0.orig/hwy/detect_targets.h
+++ highway-1.3.0/hwy/detect_targets.h
@@ -29,7 +29,7 @@
 // #define HWY_BASELINE_TARGETS (HWY_SSE4 | HWY_SCALAR)
 
 // Uncomment to override the default blocklist:
-// #define HWY_BROKEN_TARGETS HWY_AVX3
+#define HWY_BROKEN_TARGETS (HWY_AVX10_2 | HWY_AVX3_SPR | HWY_AVX3_ZEN4 | HWY_AVX3_DL | HWY_AVX3)
 
 // Uncomment to definitely avoid generating those target(s):
 // #define HWY_DISABLED_TARGETS HWY_SSE4
